spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -75,6 +96,16 @@ public Subscriber subscribe(String subscription, MessageReceiver messageReceiver
 		return subscriber;
 	}
 
+	@Override
+	public <T> Subscriber subscribeAndConvert(String subscription,
+			Consumer<ConvertedBasicAcknowledgeablePubsubMessage<T>> messageConsumer, Class<T> payloadType) {
+		return this.subscribe(subscription,
```

Comment:
We don't use `this.` for method and you are still missing a `Assert.notNull(messageConsumer)`